engines/ultima/detection.cpp

```diff
@@ -31,7 +31,7 @@ namespace Ultima {
 
 static const PlainGameDescriptor ULTIMA_GAMES[] = {
 #ifndef RELEASE_BUILD
-	{ "ultima1", "Ultima I - The First Age of Darkness" },
+	{ "ultima1", "Ultima I" }, // also called Ultima I - The First Age of Darkness
```

Comment:
I'd keep the complete title (which was used in most re-releases/collections), for consistency with the other games in the series.

Reply:
Okay, I agree with you but then what kind of naming convention should we use? The only one I could think of is to base it off of the box art because it is some kind of system. I would like to gather more opinions on this one(and to a lesser extent Laura II and I) because Honestly I don't mind either way but I think it is important for there to be some kind of standard( like the box art standard or hopefully something more clever)

Reply:
I'd prefer something which makes related series 'obvious', so if one game goes against series standard... well change that (imo)